reference.md

```diff
@@ -1005,6 +1002,39 @@ macros into smaller scopes as well. Note that macros are a
 compile-time construct; they do not exist at runtime. As such macros
 cannot be exported at the bottom of a module like functions and other values.
 
+Broadly, a macro replaces its own expression with another.
+
+Macros should return a "quoted" expression (via `` ` ``) to be used as the
+replacement. Normally that expression would "unquote" a value (via `,`) but it
+does not have to. You may not "unquote" a value outside of a "quoted"
+expression.
+
+```fennel
+
+(macro print-even [n]
+  (if (= (% n 2) 0)
+    `(print ,n)))
+
+```
+
+As a very simple example, the following macro will emit an expression that adds
+10 to a given argument. Note: this example is *intentionally simplistic* and
+`(plus-10 ...)` would be as function in pratice.
```

Comment:
Explicitly note it's a poor macro?

Reply:
I'd say we wait until @technomancy adds some notes on this section